crates/bevy_text/src/lib.rs

```diff
@@ -38,6 +38,25 @@ pub type DefaultTextPipeline = TextPipeline<Entity>;
 #[derive(Default)]
 pub struct TextPlugin;
 
+/// `TextPlugin` settings
+#[derive(Resource)]
+pub struct TextSettings {
+    /// Number of font atlases supported in a FontAtlasSet
```

Comment:
```suggestion
    /// Maximum number of font atlases supported in a FontAtlasSet
```

Reply:
Isn't that supposed to be a limit of font sizes in a font atlas and not a limit on the amount of font atlas used?

Reply:
Fixed